CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ option (BUILD_ZMQ_GATEWAY "Build zmq gateway" OFF)
 option (BUILD_VSOMEIP_GATEWAY "Build vsomeip gateway" OFF)
 option (BUILD_DDS_PROXY "Build dds proxy" OFF)
 option (BUILD_BENCH "Build nano-bench" OFF)
+option (BUILD_APP_LIB "Build library for app" OFF)
```

Comment:
What is the library for app?

Reply:
Build nanomq as a library and called by user's program